concepts/sbtc/operations/withdrawal.md

```diff
@@ -1,39 +1,25 @@
 # sBTC Withdrawal Operation
 
-The withdrawal operation allows users to convert their sBTC back to BTC. This process involves burning sBTC on the Stacks blockchain and releasing the equivalent amount of BTC on the Bitcoin blockchain.
+The sBTC withdrawal operation enables users to convert their sBTC back to BTC. This process involves burning sBTC on the Stacks blockchain and releasing an equivalent amount of BTC on the Bitcoin blockchain.
 
 ## Process Overview
 
-1. A Clarity contract call initiates the withdrawal process.
-2. The withdrawal requires 6 sortitions (Stacks block confirmations) for completion.
-3. After confirmation, sBTC Bootstrap Signers create the withdrawal transaction on the Bitcoin network.
-4. The user receives BTC in their specified Bitcoin address.
+The withdrawal process begins when a user initiates a Clarity contract call. This call triggers a series of events that culminate in the user receiving BTC in their specified Bitcoin address. The process requires six sortitions, which are essentially Stacks block confirmations, to complete. After these confirmations, sBTC Bootstrap Signers create the withdrawal transaction on the Bitcoin network.
```

Comment:
i think that can be simplified to someting like "This process requires at least six Stacks block confirmations..."